tests/components/SInputBase.spec.ts

```diff
@@ -29,7 +27,7 @@ describe('components/SInputBase', () => {
       }
     })
 
-    expect(wrapper.find('.tip').text()).toBe('Info in slot.')
```

Comment:
`find` doesn't work with teleported targets, so using `querySelector` directly. `findComponent` works, but the teleported stuff inside tooltip needs to be a separate component for it to work.